__tests__/app.test.js

```diff
@@ -33,4 +33,29 @@ describe("GET api/topics", () => {
                 });
         });
     });
-});
\ No newline at end of file
+});
+
+describe.only("GET /api/articles/:article_id/comments", () => {
+    describe("Connecting to path", () => {
+        test("Should return status 200", () => {
+            return request(app).get("/api/articles/1/comments").expect(200)
+        });
+        test("Should return the correct comment data for a specific article", () => {
+            return request(app).get("/api/articles/1/comments").expect(200)
+                .then(({ body: { comments } }) => {
+                    expect(comments[0]).toHaveProperty("comment_id", expect.any(Number))
+                    expect(comments[0]).toHaveProperty("votes", expect.any(Number))
+                    expect(comments[0]).toHaveProperty("created_at", "2020-11-03T21:00:00.000Z")
+                    expect(comments[0]).toHaveProperty("author", expect.any(String))
+                    expect(comments[0]).toHaveProperty("body", expect.any(String))
+                    expect(comments[0]).toHaveProperty("article_id", expect.any(Number))
+                });
+        });
+        test("Should return comments sorted by date in DESC order", () => {
+            return request(app).get("/api/articles/1/comments").expect(200)
+                .then(({ body: { comments } }) => {
+                    expect(comments).toBeSortedBy("created_at", { descending: true })
+            })
```

Comment:
Please add a test where the article_id exists but there are no comments for that article